client/components/DateSelector/options.js

```diff
@@ -0,0 +1,52 @@
+import { DateUtils } from 'react-day-picker';
+
+const today = new Date();
+
+const oneMonthBack = DateUtils.addMonths(new Date(), -1);
+const sixMonthsBack = DateUtils.addMonths(new Date(), -6);
+const twelveMonthsBack = DateUtils.addMonths(new Date(), -12);
+const startOfThisYear = new Date(`1/1/${today.getFullYear()}`);
+const oneWeekBack = new Date(new Date().setDate(today.getDate() - 7));
+
+const getMonthsLongName = (...dates) => {
+  const months = dates.map(date => date.toLocaleString('default', {
+    month: 'long',
+  }));
+  return months.join(' - ');
+};
+
+const formatDateToMMDD = (...dates) => {
+  const localeStringWithoutYear = dates
+    .map(date => date.toLocaleDateString('en-Us').split('/'))
+    .map(dateArr => {
+      const [day, month] = dateArr;
+      return [day, month].join('/');
+    });
+
+  return localeStringWithoutYear.join(' - ');
+};
+
+const options = [
+  {
+    text: `Last Week ( ${formatDateToMMDD(today, oneWeekBack)})`,
+    dates: { to: today, from: oneWeekBack },
+  },
+  {
+    text: `Last Month (${getMonthsLongName(oneMonthBack)})`,
```

Comment:
Let's ask UI/UX about the wording of some of these options. It seems weird for the option to say `December` if the true range for this option as of today is `12/21/2020 - 1/21/2021`

Reply:
Sure. Not sure about the implementation though.